src/datatypes/CardDetails.ts

```diff
@@ -1,13 +1,7 @@
-export type ColorCategory =
-  | 'White'
-  | 'Blue'
-  | 'Black'
-  | 'Red'
-  | 'Green'
-  | 'Colorless'
-  | 'Multicolor'
-  | 'Hybrid'
-  | 'Lands';
+
+export const COLOR_CATEGORIES = ['White', 'Blue', 'Black', 'Red', 'Green', 'Colorless', 'Multicolored', 'Hybrid', 'Lands'] as const;
```

Comment:
Using typescript const assertions so that we can use COLOR_CATEGORIES to check if a string is within its set of values. Could not see how to do that using the ColorCategory type itself.